bdd/features/tipo8/_transaction.feature

```diff
@@ -0,0 +1,31 @@
+Feature: A transaction is generated, authorized and confirmed
+
+  Background:
+    Given the initiative is "Scontoditipo8"
+    And the merchant is qualified
```

Comment:
Missing: 
- merchant 1 
- And the citizen A is 25 years old at most (and for B)